### ISSUE TITLE
Warning for line-length should be at 140 chars

### ISSUE THREAD

**Issue by andrekeller** [rodjek/puppet-lint#396]
According to:
https://docs.puppetlabs.com/guides/style_guide.html#spacing-indentation-and-whitespace

Line length should not exceed 140 chars, puppet-lint currently warns if length exceeds 80 chars

**Comment by ghoneycutt** [comment]
The Puppet Labs Style Guide is a versioned document and some people may comply with the 80 character limit, while others the 140 in the newest version of the guide (while most of us ignore this... though that's a separate rant). Suggest that a separate check for 140 characters be added.

**Comment by jonbrouse** [comment]
If the style guide changes shouldn't the lint checker rules change? If a user decides they want to continue to use the 80 character limit then they can adjust default behavior of the rake task. Not the other way around. +1 for updating the line length max to 140 chars.

**Comment by ghoneycutt** [comment]
TL;DR We should have an additional check for 140 characters and make that the default, which I think satisfies your objective.

The linter should change to keep up though this is a question of dealing with multiple versions of the style guide, especially in light of the needed change to Puppet v4 and the transition time when people are dealing with Puppet v3 and v4 and their respective style guides. I've outlined an approach in #398

**Comment by strider** [comment]
:+1: for @ghoneycutt comment

**Comment by divansantana** [comment]
The [documentation](http://puppet-lint.com/checks/140chars/) is updated to say 140chars.
Seems there is [code](https://github.com/rodjek/puppet-lint/commit/2b48ab36bb5334a41f98f8bd75867cd69eb6f859) for this too and was merged #419 

Can we please get an updated stable release so this can be closed.

**Comment by rnelson0** [comment]
Closing this as the change was merged in #419 and a new release is tracked in #443. Please add a comment there if you'd like.
